packages/clerk-js/src/ui/components/OrganizationProfile/OrganizationSettings.tsx

```diff
@@ -131,10 +133,11 @@ const OrganizationDangerSection = () => {
           colorScheme='danger'
           textVariant='buttonExtraSmallBold'
           onClick={() => navigate('leave')}
-          isDisabled={isAdmin && !hasMoreThanOneAdmin}
+          // TODO: rewrite to check if user has all clerk permissions and there are more than 1 admins
+          isDisabled={!!canDeleteOrganization && !hasMoreThanOneAdmin}
```

Comment:
We can drop the `hasMoreThanOneAdmin` check and rely on the BE in order to allow or block this action. It's good enough for now to keep it disabled if setting is not enabled or the user doesn't have the proper permission

Reply:
Correct, this is a leftover, will update it asap